script.js

```diff
@@ -10,13 +10,14 @@ const winResults = [
 ];
 const symbols = {"player1": "X", "player2":"O"};
 
-var isPlayer1, board;
-var player1Status = [];
-var player2Status = [];
-var allSquares = document.getElementsByClassName("square");
-var startButton = document.getElementById("startButton");
-var clearButton = document.getElementById("clearButton");
-var infoP = document.getElementById("info");
+let isPlayer1;
+let board;
+let player1Status = [];
+let player2Status = [];
+const allSquares = document.getElementsByClassName("square");
+const startButton = document.getElementById("startButton");
+const clearButton = document.getElementById("clearButton");
+const infoP = document.getElementById("info");
```

Comment:
We're not reassigning those variables. We can use `const`

Reply:
so in es6, avoid use "var"?

Reply:
yes :)